docs/docs/v7-upgrade-guidance.mdx

```diff
@@ -303,6 +303,10 @@ Remove the references to the option `schemaXml` from your scripts. Verify that y
 
 In the previous version of CLI for Microsoft 365, you could get deleted users in Azure AD using the command `aad user list` and providing an option `deleted`. In the latest release, we have removed that option from the command. To get all deleted users in your tenant, you can now use the command [aad user recyclebinitem list](./cmd/aad/user/user-recyclebinitem-list.mdx).
 
+## Removed `deleted` option from `aad group list` command
```

Comment:
Currently, you inserted your section right in the middle of another section. Notice that every change in this file has a title and a subtitle (what action do I need to take?).
So let's move it and add a "what action do I need to take" subsection.